app/policies/admin/site_notice_policy.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::SiteNoticePolicy < ApplicationPolicy
+
+  class Scope < Scope
+    def resolve
+      all.order('updated_at desc')
```

Comment:
I think this isn't even used, but in any case: the ordering shouldn't be part of the authorization scope.